### PR TITLE
feat(cross): #794 calculate scorecard feature

### PR DESCRIPTION
**Pull request by jpverde**
- Create arg `calculateScorecard`
- Create module `calculateScorecard`
- Add documentation for `calculateScorecard` arg and module
- Use scorecard module on makes

### CODE REVIEW

File: makes.nix [hunk 1]
```diff
@@ -13,6 +13,10 @@
       pubKey = "makes.cachix.org-1:HbCQcdlYyT/mYuOx6rlgkNkonTGUjVr3D+YpuGRmO+Y=";
     };
   };
+  calculateScorecard = {
+    enable = true;
+    target = "github.com/fluidattacks/makes";
```
Comment (dsalaza4) [inline makes.nix:18]: is it possible to run on a local target? That way the check can actually review the current change and not the main branch only.

Comment (jpverde) [reply makes.nix:18]: By local you mean the author branch? or a local directory?

Comment (dsalaza4) [reply makes.nix:18]: A local directory

Comment (jpverde) [reply makes.nix:18]: #794 I will be commenting updates about this on the issue.

**Pull request merged**